src/harness/fourslash.ts

```diff
@@ -1740,31 +1722,25 @@ namespace FourSlash {
                 }

```

Comment:
`this.applyEdits` returns the number of characters added/removed in the file, but this is different from the adjustment to the caret position (which is the number of characters added/removed in the file *before the caret*).